labkey/experiment.py

```diff
@@ -218,6 +218,55 @@ def save_batches(
     return None
 
 
+def lineage(
+    server_context: ServerContext,
+    lsids: List[str],
+    children: bool = None,
+    container_path: str = None,
+    cpas_type: str = None,
+    exp_type: str = None,
+    depth: int = None,
+    parents: bool = None,
+    run_protocol_lsid: str = None,
+):
+    """
+    :param server_context: A LabKey server context. See utils.create_server_context.
+    :param lsids: Array of LSIDs for the seed ExpData, ExpMaterials, or ExpRun
+    :param children: Include children in the lineage response. Defaults to true.
+    :param container_path: labkey container path if not already set in context
+    :param cpas_type: Optional LSID of a SampleSet or DataClass to filter the response. Defaults to include all.
+    :param exp_type: Optional experiment type to filter response -- either "Data", "Material", or "ExperimentRun".
+    Defaults to include all.
+    :param depth: An optional depth argument. Defaults to include all.
+    :param parents: Include parents in the lineage response. Defaults to true.
+    :param run_protocol_lsid: Optional Exp Run Protocol Lsid to filter response. Defaults to include all.
+    """
```

Comment:
in the Rlabkey version, I also had 3 boolean params. Were those intentionally left off?
- includeProperties: (boolean) include properties in the lineage response
- includeInputsAndOutputs: (boolean) include inputs and outputs in the lineage response
- includeRunSteps: (boolean) include run steps in the lineage response

Reply:
I only took the params we used in our JS client code, but I can add those here easily

Reply:
Done